WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -94,6 +94,8 @@ final class OrdersRootViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
```

Comment:
It seems that we're missing a super call here 🤔 

```suggestion
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
```

Reply:
nice catch! added in https://github.com/woocommerce/woocommerce-ios/pull/5715/commits/6e3c99e5c90eca1fd7f5800afc1ba178a7e5ba42